scripts/libs.versions.toml

```diff
@@ -8,10 +12,11 @@ androidx-benchmark-junit4 = "1.1.0-beta04"
 androidx-constraintlayout = "1.0.1"
 androidx-corektx = "1.8.0"
 androidx-lifecycle-compose = "2.5.1"
+androidx-lifecycle-runtime-compose = "2.6.0-alpha02" # merge with above when not alpha
 androidx-navigation = "2.5.1"
 androidx-palette = "1.0.0"
-androidx-test = "1.4.0"
-androidx-test-espresso = "3.4.0"
+androidx-test = "1.3.0"
+androidx-test-espresso = "3.3.0"
```

Comment:
Intentional downgrade?

Reply:
I was getting annoying dependency clashes in one of the samples, but I will fix this before merging

Reply:
OK it seems compose 1.3 beta03 pins to these versions so we will have to go with this for now. I assume this will be upgraded again in the next release.